backend/src/test/java/com/ody/mate/service/MateServiceTest.java

```diff
@@ -105,4 +115,69 @@ void findAllByMemberAndMeetingIdException() {
         assertThatThrownBy(() -> mateService.findAllByMemberAndMeetingId(member2, meeting.getId()))
                 .isInstanceOf(OdyBadRequestException.class);
     }
+
+    @DisplayName("Eta상태가 지각 위기인 mate를 콕 찌를 수 있다")
+    @Test
+    void nudgeSuccessWhenLateWarning() {
+        LocalDateTime now = TimeUtil.nowWithTrim();
+        Meeting meeting = new Meeting(
+                "오디",
+                now.toLocalDate(),
+                now.toLocalTime().plusMinutes(1L),
+                Fixture.TARGET_LOCATION,
+                "초대코드"
```

Comment:
[제안]
치명적인 부분은 아니지만 코드 라인을 줄일 수 있으면 줄이는 편이라 제안드립니다

meeting 생성시 초단위가 제거 되고 있어 
LocalDateTime now = TimeUtil.nowWithTrim();를 선언하고 사용하지 않아도 될 것 같아요

```suggestion
        Meeting meeting = new Meeting(
                "오디",
                LocalDate.now(),
                LocalTime.now().plusMinutes(1L),
                Fixture.TARGET_LOCATION,
                "초대코드"
```

Reply:
private method로 특정 시간 대 meeting 생성 로직을 크로스 커팅해주었습니다!

```java
private Meeting makeSavedMeetingByTime(LocalDateTime time) {
        Meeting meeting = new Meeting(
                "오디",
                time.toLocalDate(),
                time.toLocalTime(),
                Fixture.TARGET_LOCATION,
                "초대코드"
        );
        return meetingRepository.save(meeting);
    }
```